pkg/distro/rhel/rhel9/azure.go

```diff
@@ -319,8 +319,12 @@ func azureInternalBasePartitionTables(t *rhel.ImageType) (disk.PartitionTable, b
 // IMAGE CONFIG
 
 // use loglevel=3 as described in the RHEL documentation and used in existing RHEL images built by MSFT
-func defaultAzureKernelOptions() []string {
-	return []string{"ro", "loglevel=3", "console=tty1", "console=ttyS0", "earlyprintk=ttyS0", "rootdelay=300"}
+func defaultAzureKernelOptions(rd *rhel.Distribution) []string {
+	kargs := []string{"ro", "loglevel=3", "console=tty1", "console=ttyS0", "earlyprintk=ttyS0", "rootdelay=300"}
+	if rd.IsRHEL() && common.VersionGreaterThanOrEqual(rd.OsVersion(), "9.6") {
+		kargs = append(kargs, "nvme_core.io_timeout")
```

Comment:
Should be

```suggestion
		kargs = append(kargs, "nvme_core.io_timeout=240")
```

per https://github.com/osbuild/images/issues/1421

Reply:
Yup.  Stupid mistake.  Thanks for catching it!